src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +240,30 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display notices for unsupported currencies.
+	 *
+	 * @since 5.5.5
+	 *
+	 * @return void
+	 */
+	public function maybe_display_unsupported_currency_notice() {
+		if ( Currency::is_supported_currency() ) return;
```

Comment:
```suggestion
		if ( Currency::is_supported_currency() ) {
			return;
		}
```
We need to add the braces for IFs statements.

We're using the `supported_currency` logic as if we're only going to have one unsupported currency. If we're going this road we need to modify the logic to be able to iterate over the unsupported currencies and make a notice for each.

Reply:
Added braces for IF statement

Reply:
@juanfra I'm not sure if we will need to iterate over each unsupported currency and make a notice for each one. Since the user will only be selecting one currency setting per time, we wouldn't need to make a notice for each one, except for the one the user selects that falls into the unsupported category.